.github/workflows/ci.yml

```diff
@@ -513,10 +513,6 @@ jobs:
           java-version: 11
           cache: sbt
 
-      - name: sbt update
-        if: matrix.java == 'temurin@11' && steps.setup-java-temurin-11.outputs.cache-hit == 'false'
-        run: sbt '++ ${{ matrix.scala }}' reload +update
```

Comment:
So the issue is that it's still trying to cache like sbt. I think we should have a helper to generate a java-setup step with caching disabled. That would remove this parameter and also the `sbt update` step.

Reply:
oh hm, let me look at that

Reply:
Alright, let's see how you like this version :D